packages/common/src/pipes/number_pipe.ts

```diff
@@ -17,59 +17,76 @@ import {invalidPipeArgumentError} from './invalid_pipe_argument_error';
  * @ngModule CommonModule
  * @description
  *
- * Transforms a number into a string,
- * formatted according to locale rules that determine group sizing and
- * separator, decimal-point character, and other locale-specific
- * configurations.
- *
- * If no parameters are specified, the function rounds off to the nearest value using this
- * [rounding method](https://en.wikibooks.org/wiki/Arithmetic/Rounding).
- * The behavior differs from that of the JavaScript ```Math.round()``` function.
- * In the following case for example, the pipe rounds down where
- * ```Math.round()``` rounds up:
- *
- * ```html
- * -2.5 | number:'1.0-0'
- * > -3
- * Math.round(-2.5)
- * > -2
- * ```
+ * Formats a value according to digit options and locale rules.
+ * Locale determines group sizing and separator,
+ * decimal-point character, and other locale-specific configurations.
  *
  * @see `formatNumber()`
  *
  * @usageNotes
- * The following code shows how the pipe transforms numbers
- * into text strings, according to various format specifications,
- * where the caller's default locale is `en-US`.
+ *
+ * ### digitsInfo
+ *
+ * The value's decimal representation is specified by the `digitsInfo`
+ * parameter, written in the following format:<br>
+ * `"{minIntegerDigits}.{minFractionDigits}-{maxFractionDigits}"`.
```

Comment:
```suggestion
 * ```
 * {minIntegerDigits}.{minFractionDigits}-{maxFractionDigits}"`
 * ```
```

Reply:
@aikidave That was my first take also, actually. Putting it in a code block. @petebacondarwin then suggested changing that to what it is now. IMO I it looks better now, but i'll defer to what you guys decide : )

Reply:
For reference here are the two different layouts:

**With the inline code (https://pr40714-ff9da13.ngbuilds.io/api/common/DecimalPipe#digitsinfo)**

<img width="812" alt="Screenshot 2021-02-17 at 09 14 52" src="https://user-images.githubusercontent.com/15655/108182184-a78e6d80-7100-11eb-8c9f-779267cd6540.png">

**With the block of code (https://pr40714-a3a111b.ngbuilds.io/api/common/DecimalPipe#description):**

<img width="843" alt="Screenshot 2021-02-17 at 09 14 43" src="https://user-images.githubusercontent.com/15655/108182188-a8bf9a80-7100-11eb-9173-981a96eebcd7.png">

Reply:
I personally like it much better as code block. You don't have the quotation marks, for one thing, and it lets the format stand out from the rest of the paragraph.

Reply:
What's the protocol for settling differences of opinion at Google? Roshambo? ; )

Reply:
@aikidave is in charge of the docs, so I bow to his suggestion.

Reply:
I say go with the code block then! Thanks!

Reply:
> @aikidave is in charge of the docs, so I bow to his suggestion.

Magnanimous : )
Just pushed update. Thanks!